### PR TITLE
Always show accuracy in play-details

### LINKED ISSUE #1146: Top ranks on profile are missing accuracy display
I think we need to get this back in the design.

image

### PR DESCRIPTION
**Pull request by notbakaneko**
resolves #1146 

There's already logic to show the accuracy if there's no `score.weight` or `score.pp`; this just makes it show all the time.
It might look kind of unbalanced, though?

image

### CODE REVIEW

**Comment by arflyte** [comment]
Increase the padding between pp and weighted, and reduce the line height between weighted and accuracy

File: resources/assets/coffee/react/_components/play-detail.coffee [hunk 1]
```diff
@@ -73,17 +73,15 @@ el = React.createElement
                   osu.trans('users.show.extra.top_ranks.pp', amount: Math.round(score.pp))
                 else
                   score.score.toLocaleString()
-            if score.weight
-              div
-                className: 'detail-row__detail-row detail-row__detail-row--bottom'
-                span
+            div
+              className: 'detail-row__detail-row detail-row__score-details'
```
Comment (nanaya) [inline resources/assets/coffee/react/_components/play-detail.coffee]: more like `detail-row__detail-row--score-details`

**Comment by notbakaneko** [comment]
now with 100% less bad renderer
image

**Comment by notbakaneko** [comment]
I've just noticed the mod icons don't show on Safari? ಠ_ಠ

**Comment by notbakaneko** [comment]
Safari doesn't consider the background-image as part of its content sizing, so the height of the mod icons block in Safari is 0 🎉

**Comment by notbakaneko** [comment]
separate issue for missing mod icons https://github.com/ppy/osu-web/issues/1256

**Pull request merged**